vispy/app/application.py

```diff
@@ -77,6 +77,19 @@ def process_events(self):
         """
         return self._backend._vispy_process_events()
 
+    def sleep(self, duration_sec):
+        """
+        Sleep for the given duration. This is used to reduce
+        CPU stress when VisPy is run in interactive mode.
+        see inputhook.py for details
```

Comment:
Also, perhaps `sleep()` should be a private method since we only use it internally and (I suspect) would not be a useful public feature (and if it is, we can make it public later).